server/routes.go

```diff
@@ -732,6 +732,11 @@ func (s *Server) ListModelsHandler(c *gin.Context) {
 			}
 
 			n := model.ParseNameFromFilepath(rel)
```

Comment:
We should consider having `Parse` and `Parse*` functions return an error like `url.Parse` and similar functions in the go standard lib?